### PR TITLE
Sarkars/grappler pass script

### PR DESCRIPTION
**Pull request by sayantan-nervana**
Pytest for tf2ngraph

### CODE REVIEW

File: tools/create_small_graph.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import tensorflow as tf
```
Comment (sayantan-nervana) [inline tools/create_small_graph.py]: TODO: remove this file

File: test/python/test_convert_script.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+                              commandline):
+        if out_format == 'savedmodel':
+            # TODO: enable this
+            return
+        assert TestConversionScript.format_and_loc_match(inp_format, inp_loc)
+        out_loc = inp_loc.split('.')[0] + '_modified' + (
+            '' if out_format == 'savedmodel' else ('.' + out_format))
+        print('_' * 50)
+        print(inp_format, inp_loc, out_format, commandline, out_loc)
+        print('_' * 50)
```
Comment (sayantan-nervana) [inline test/python/test_convert_script.py]: TODO: remove prints here

File: CMakeLists.txt [hunk 3]
```diff
@@ -286,6 +286,7 @@ add_dependencies(ngraph_lib ext_ngraph)
 add_subdirectory(third-party)
 add_subdirectory(logging)
 add_subdirectory(src)
+add_subdirectory(tools)
```
Comment (sayantan-nervana) [inline CMakeLists.txt]: Need `tf2ngraph` script in the artifacts directory

File: test/ci/buildkite/ngtf-cpu_centos.yaml [hunk 4]
```diff
@@ -78,7 +78,7 @@
   - command: |
       source /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID/venv/bin/activate 
       pip install pytest
-      PYTHONPATH=`pwd` python3 test/ci/buildkite/test_runner.py \
+      PYTHONPATH=`pwd`:`pwd`/tools python3 test/ci/buildkite/test_runner.py \
```
Comment (sayantan-nervana) [inline test/ci/buildkite/ngtf-cpu_centos.yaml]: Need `tools` to access `command_executor`

File: tools/tf2ngraph.py [hunk 5]
```diff
@@ -197,7 +193,6 @@ def convert(inp_format, inp_loc, out_format, out_loc, outnodes):
    """
     assert inp_format in allowed_formats['input']
     assert out_format in allowed_formats['output']
-    assert ngraph_bridge.is_grappler_enabled()
```
Comment (sayantan-nervana) [inline tools/tf2ngraph.py]: TODO: this assert should probably stay. Also add an assert in the pytest that the converted model has an ngraphencapsulate op in it

File: test/python/test_convert_script.py [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+        implies = lambda p, q: (not p) or (p and q)
+        #if its pbtxt, file name has pbtxt AND if its savedmodel, file name does not have pbtxt
+        return implies(
+            format == 'pbtxt', 'pbtxt' == loc.split('.')[-1]) and implies(
+                format == 'savedmodel', 'pbtxt' != loc.split('.')[-1])
+
+    @pytest.mark.parametrize(('commandline'),
+                             (True,))  #TODO: add False for functional api test
+    @pytest.mark.parametrize(('inp_format', 'inp_loc'), (
+        ('pbtxt', 'sample_graph.pbtxt'),
```
Comment (sindhu-nervana) [inline test/python/test_convert_script.py]: do you want to call this `sample_graph_device.pbtxt` to differentiate?

**Pull request merged**